src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -1997,4 +2206,9 @@ private HyperwalletPayPalAccount copy(HyperwalletPayPalAccount payPalAccount) {
         return payPalAccount;
     }
 
+    private HyperwalletVenmoAccount copy(HyperwalletVenmoAccount venmoAccount) {
+        venmoAccount = HyperwalletJsonUtil.fromJson(HyperwalletJsonUtil.toJson(venmoAccount), HyperwalletVenmoAccount.class);
```

Comment:
we can directly return from here if we are not going to use `venmoAccount` variable